test/e2e/resilience/resilience_multizone_universal_postgres.go

```diff
@@ -0,0 +1,115 @@
+package resilience
+
+import (
+	"strings"
+
+	"github.com/kumahq/kuma/pkg/config/core"
+	. "github.com/kumahq/kuma/test/framework"
+	"github.com/kumahq/kuma/test/framework/deployments/postgres"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+)
+
+func ResilienceMultizoneUniversalPostgres() {
+	var global, remote_1 Cluster
+	var optsGlobal, optsRemote1 []DeployOptionsFunc
+
+	BeforeEach(func() {
+		clusters, err := NewUniversalClusters(
+			[]string{Kuma1, Kuma2},
+			Silent)
+		Expect(err).ToNot(HaveOccurred())
+
+		// Global
+		global = clusters.GetCluster(Kuma1)
+		optsGlobal = []DeployOptionsFunc{}
+		err = NewClusterSetup().
+			Install(postgres.Install()).
+			Install(Kuma(core.Global, optsGlobal...)).
+			Setup(global)
+		Expect(err).ToNot(HaveOccurred())
+		err = global.VerifyKuma()
+		Expect(err).ToNot(HaveOccurred())
+
+		globalCP := global.GetKuma()
+
+		// Cluster 1
+		remote_1 = clusters.GetCluster(Kuma2)
+		optsRemote1 = []DeployOptionsFunc{
+			WithGlobalAddress(globalCP.GetKDSServerAddress()),
+		}
+
+		err = NewClusterSetup().
+			Install(postgres.Install()).
+			Install(Kuma(core.Remote, optsRemote1...)).
+			Setup(remote_1)
+		Expect(err).ToNot(HaveOccurred())
+		err = remote_1.VerifyKuma()
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	E2EAfterEach(func() {
+		err := remote_1.DeleteKuma(optsRemote1...)
+		Expect(err).ToNot(HaveOccurred())
+		err = remote_1.DismissCluster()
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
nit: here and above, IIUC the normal ginkgo pattern would be
```suggestion
		Expect(remote_1.DismissCluster()).Should(Succeed())
```

Reply:
Thank you for suggestion, I like this pattern, did it for consistency, but I think we could start introducing this pattern in our tests